pkg/agent/route/route.go

```diff
@@ -310,10 +322,120 @@ func (c *Client) RemoveServiceRouting() error {
 	ipRule := netlink.NewRule()
 	ipRule.IifName = c.nodeConfig.GatewayConfig.Name
 	ipRule.Mark = iptables.RtTblSelectorValue
+	ipRule.Mask = iptables.RtTblSelectorValue
 	ipRule.Table = AntreaServiceTableIdx
 	ipRule.Priority = AntreaIPRulePriority
 	if err = netlink.RuleDel(ipRule); err != nil {
 		return fmt.Errorf("ip rule delete: %w", err)
 	}
 	return nil
 }
+
+// resolveDefaultRouteNHMAC resolves the MAC of default route next
+// hop on service route table.
+func (c *client) resolveDefaultRouteNHMAC() (net.HardwareAddr, error) {
+	// This MAC is relevant to L2 mode only.
+	// TODO, for now uses AKS default MAC.
+	return net.ParseMAC("12:34:56:78:9a:bc")
```

Comment:
Why we use AKS MAC? Maybe define a random one.
Use a constant.

Reply:
in l3 case, no reason, it can be anything. It matter in L2 (AKS case) where only known MAC may be used for next hop.